src/index.js

```diff
@@ -43,7 +43,7 @@ const customFormat = printf(({ level, message, timestamp, context }) => {
     message = util.inspect({
```

Comment:
Only problem that remains with this, is that the output is not valid JSON, which will be hard to parse.